policyengine_us/variables/household/demographic/tax_unit/tax_unit_stillborn_parent.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class tax_unit_stillborn_parent(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Is the parent of a stillborn child"
```

Comment:
Ever or stillborn in the filing year?
```suggestion
    label = "Head or spouse is the parent of a stillborn child"
```